src/iter/mod.rs

```diff
@@ -0,0 +1,93 @@
+// Written in 2023 by Andrew Poelstra <apoelstra@wpsoftware.net>
```

Comment:
Hey @apoelstra, I saw this line and wondered if you added it as a habit or if I have been annoying you by removing them? Or was this PR just before we discussed all that? Cheers.

Reply:
I think it predates your PR to remove these. I would've just mildlessly copied the line from other files.

I'm certainly not annoyed :).

Reply:
Sweet man, cheers.